homeassistant/components/tasmota/discovery.py

```diff
@@ -45,6 +45,50 @@ async def async_start(
 ) -> bool:
     """Start MQTT Discovery."""
 
+    async def _load_platform(platform):
```

Comment:
Docstring is missing.

Reply:
Do we require docstrings on private functions?

Reply:
Yes.

Reply:
OK, thanks, fixed now.

Maybe we should enable the check for docstrings on all functions?
One suggestion seems to be to set pylint option `no-docstring-rgx=__.*__`